python_modules/dagster/dagster_tests/definitions_tests/declarative_automation_tests/daemon_tests/test_e2e.py

```diff
@@ -547,3 +583,24 @@ def test_custom_condition() -> None:
             _execute_ticks(context, executor)
             runs = _get_runs_for_latest_ticks(context)
             assert len(runs) == 0
+
+
+def test_massive_user_code(capsys) -> None:
+    with get_grpc_workspace_request_context(
+        "massive_user_code"
+    ) as context, get_threadpool_executor() as executor:
+        freeze_dt = datetime.datetime(2024, 8, 16, 1, 35)
+
+        for _ in range(2):
+            clock_time = time.time()
+            with freeze_time(freeze_dt):
+                _execute_ticks(context, executor)
+                runs = _get_runs_for_latest_ticks(context)
+                assert len(runs) == 0
+            duration = time.time() - clock_time
+            assert duration < 40.0
+
+            freeze_dt += datetime.timedelta(minutes=1)
+
+    for line in capsys.readouterr():
```

Comment:
nit: cross referencing something besides just capsys might be nice

Reply:
Updated the test to grab the tick and make sure it's successful, which will catch other sorts of errors as well